src/styles/isomer-cms/elements/base.scss

```diff
@@ -41,7 +41,6 @@ li {
 .wrapper {
   height: calc(100vh - #{$header-height});
   width: 100%;
-  top: $header-height;
   position: relative;
```

Comment:
does this change not affect the styling?

Reply:
It does - but it's necessary because it fixes badly written CSS. 

The `.header` element was previously set to `position: fixed`, while the `.wrapper` element is `position: relative`. And so the janky `top: $header-height;` was necessary to move the `.wrapper` element down to prevent overlap with the header.

Now that we are setting `.header` to `position: relative` (in order to accommodate the banner), we can remove that janky `top: $header-height;` from `.wrapper`.